regression/tests/lms/test_instructor_dashboard.py

```diff
@@ -42,17 +40,3 @@ def setUp(self):
         self.dashboard_page.select_course(get_course_display_name())
         self.course_page.wait_for_page()
         self.instructor_dashboard.visit()
-
-    @skipIf(
-        LMS_BASE_URL != LMS_STAGE_BASE_URL,
-        "Url can't be tested on sandbox"
-    )  # LT-61
-    def test_analytics_link(self):
-        """
-        Verifies that edX Insights link is clicked and displayed
-        """
-        self.instructor_dashboard.click_analytics_tab()
-        self.assertEquals(
-            self.instructor_dashboard.get_insights_title_text(),
-            'INSIGHTS'
```

Comment:
The insights courses page (page you land on after logging into Insights) has been taking longer to load recently. I don't know what the default timeout is, but maybe increasing it > 15s will solve the flakiness.